core/fslock/fslock_error.cpp

```diff
@@ -0,0 +1,17 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "fslock/fslock_error.hpp"
+
+OUTCOME_CPP_DEFINE_CATEGORY(fc::fslock, FSLockError, e) {
+  using fc::fslock::FSLockError;
+
+  switch (e) {
+    case (FSLockError::FILE_LOCKED):
+      return "FileLock: file is locked";
+    default:
+      return "FileLock: unknown error";
```

Comment:
pls add case for each option and remove the 'default' section,
It pursues following goal: if you forget to consider a case, clang-tidy fails with error.
If you make the 'default' section, you can unintentionally pass an error.

Reply:
`warning: control reaches end of non-void function`
If it is normal, then OK

Reply:
It means that you checked not all execution paths.